src/Raven.Server/Documents/Handlers/RevisionsHandler.cs

```diff
@@ -47,6 +47,13 @@ public async Task Revert()
                 await processor.ExecuteAsync();
         }
 
+        [RavenAction("/databases/*/revisions/revert/document", "POST", AuthorizationStatus.ValidUser, EndpointType.Write)]
```

Comment:
Regarding `/document`, this endpoint allows to revert multiple documents, not a single one, right? I wonder whether we need `/document` part at all? Is is possible we might have another `/revisions/revert/XXX` in the future?

Reply:
We already have `/databases/*/revisions/revert/` so it needs to be different.

Reply:
let's align it with other endpoint names so instead of `/document` let's do `/docs`

Reply:
Done